.github/workflows/tests.yml

```diff
@@ -0,0 +1,13 @@
+name: Run tests
+on: [pull_request]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.19'
+      - uses: actions/checkout@v3
+      - name: test
+        run: go test -v ./...
```

Comment:
@inelpandzic please check results

Reply:
These unit tests are currently failing, need to fix them.

Reply:
[e3e1bf9](https://github.com/percona/percona-server-mysql-operator/pull/233/commits/e3e1bf94d9acdd0dcc27e13cdb49046b41053f4d)